src/App.js

```diff
@@ -1,10 +1,54 @@
+import styled, { createGlobalStyle } from 'styled-components';
+
+import Todo from './components/Todo';
 
 function App() {
   return (
-    <div>
-      <h1>15기 화이팅~</h1>
-    </div>
+    <Wrapper>
+      <GlobalStyle />
+      <Main>
+        <MainTitle>할 일 목록</MainTitle>
+        <Todo />
+      </Main>
+    </Wrapper>
   );
 }
 
+const Wrapper = styled.div`
+  width: 100%;
+`;
+
+const GlobalStyle = createGlobalStyle`
+  *{  
+    box-sizing: border-box;
+    font-family: 'Noto Sans KR', sans-serif;
+  }
+  html {
+    font-size : 16px;
+  }
+  body {
+    background-color : #F6FFDB;
```

Comment:
그냥 `background`로 하셔도 됩니다!

```suggestion
    background: #F6FFDB;
```